examples/server/main.go

```diff
@@ -0,0 +1,148 @@
+	r.HandleFunc("/employees", EmployeesHandler)
+
+	staticRouter := r.PathPrefix("/static/")
+	staticRouter.Handler(http.StripPrefix("/static", http.FileServer(http.Dir("./public"))))
+
+	http.Handle("/", r)
+
+	srv := &http.Server{
+		Addr: "0.0.0.0:3000",
+		// Good practice to set timeouts to avoid Slowloris attacks.
+		WriteTimeout: time.Second * 15,
+		ReadTimeout:  time.Second * 15,
+		IdleTimeout:  time.Second * 60,
+		Handler:      r, // Pass our instance of gorilla/mux in.
+	}
+
+	// Run our server in a goroutine so that it doesn't block.
+	go func() {
+		if err := srv.ListenAndServe(); err != nil {
+			log.Println(err)
+		}
+	}()
+	log.Println("Server running on port 3000")
+
+	c := make(chan os.Signal, 1)
+	// We'll accept graceful shutdowns when quit via SIGINT (Ctrl+C)
+	// SIGKILL, SIGQUIT or SIGTERM (Ctrl+/) will not be caught.
+	signal.Notify(c, os.Interrupt)
+
+	// Block until we receive our signal.
+	<-c
+
+	// Create a deadline to wait for.
+	ctx, cancel := context.WithTimeout(context.Background(), wait)
+	defer cancel()
+	// Doesn't block if no connections, but will otherwise wait
+	// until the timeout deadline.
+	srv.Shutdown(ctx)
+	// Optionally, you could run srv.Shutdown in a goroutine and block on
+	// <-ctx.Done() if your application should wait for other services
+	// to finalize based on context cancellation.
+	log.Println("shutting down")
+	os.Exit(0)
+}
+
+// HomeHandler will be the base handler in where we will show information about
+// token and different actions you can do
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
+	var t *template.Template
+	if token == nil {
+		t, _ = template.New("index").Parse(indexTemplate)
+	} else {
+		t, _ = template.New("connected").Parse(connectedTemplate)
+	}
+	t.Execute(w, nil)
+}
+
+// StartFactorialOAuthHandler is the handler that will start the process of Auth with
+// the Factorial platform
+func StartFactorialOAuthHandler(w http.ResponseWriter, r *http.Request) {
+	http.Redirect(w, r, provider.GetAuthURL("uniq_state"), http.StatusFound)
+}
+
+// FactorialOAuthCallbackHandler is the handler in where we are going to receive a
+// successful callback with a code that can we use to get our user token
+func FactorialOAuthCallbackHandler(w http.ResponseWriter, r *http.Request) {
+	var err error
+	token, err = provider.GetTokenFromCode(r.FormValue("code"))
+	if err != nil {
+		log.Panic(err)
+	}
+	t, _ := template.New("connected").Parse(connectedTemplate)
+	t.Execute(w, token)
+}
+
+// EmployeesHandler is the handler used for get all the employees
+// and print them on a list template
+func EmployeesHandler(w http.ResponseWriter, r *http.Request) {
+	cl, err := factorial.New(
+		factorial.WithOAuth2Client(provider.Client(token)),
+	)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	employees, err := cl.ListEmployees()
+	if err != nil {
+		log.Panicln("Error while getting employees", err)
+	}
+	t, _ := template.New("employees").Parse(employeesTemplate)
```

Comment:
why you hidding the `err` here? :thinking:

Reply:
sometime I'm lazy haha because on that case if it fails I will see a blank screen so I know something went wrong, but I added anyway the error for give us more context failure!